### PR TITLE
ch 11.8 proofread; several guimenu corrections

### PR DESCRIPTION
**Pull request by cjschroder**
Re guimenus and slashes. In most circumstances it is grammatically correct to use slashes with no surrounding spaces, like either/or, red/blue, foo/fie. However, when combining GUI form fields, e.g. <guimenu>vCenter Username</guimenu> / <guimenu>vCenter Password</guimenu>, inserting spaces makes sense as an indicator that it is two separate fields. There are no grammar rules for this (that I know of), so we can do what we want. For consistency, I enclosed each field name in guimenu tags, rather than combining them. I have no preference between <guimenu>vCenter Username</guimenu> / <guimenu>vCenter Password</guimenu> and <guimenu>vCenter Username / vCenter Password</guimenu>, I just like using a single convention.

### CODE REVIEW

File: xml/depl_nodes.xml [hunk 1]
```diff
@@ -2306,7 +2297,7 @@ getent group postgres | cut -d: -f3</screen>
     </listitem>
    </varlistentry>
    <varlistentry>
-    <term><guimenu>User Name / Password for accessing the ECOM server</guimenu>
+    <term><guimenu>User Name</guimenu> / <guimenu>Password for accessing the ECOM server</guimenu>
```
Comment (taroth21) [inline xml/depl_nodes.xml]: In the Web UI, the two fields are named: 'Username for accessing the ECOM server' and 'Password for accessing the ECOM server'.

File: xml/depl_nodes.xml [hunk 2]
```diff
@@ -2381,29 +2372,28 @@ getent group postgres | cut -d: -f3</screen>
...
+    <term><guimenu>IP</guimenu> / <guimenu>Port for SMI-S</guimenu></term>
     <listitem>
      <para>
       IP address and port of the ETERNUS SMI-S Server.
      </para>
     </listitem>
    </varlistentry>
    <varlistentry>
-    <term>Username / Password for SMI-S</term>
+    <term><guimenu>Username</guimenu> / <guimenu>Password for SMI-S</guimenu></term>
```
Comment (taroth21) [inline xml/depl_nodes.xml]: In the Web UI, the fields are named: 'IP for SMI-S', 'Port for SMI-S, 'Username for SMI-S', 'Password for SMI-S'

File: xml/depl_nodes.xml [hunk 3]
```diff
@@ -2467,7 +2457,7 @@ getent group postgres | cut -d: -f3</screen>
     </listitem>
    </varlistentry>
    <varlistentry>
-    <term><guimenu>User Name/Password for Accessing NetApp</guimenu>
```
Comment (taroth21) [inline xml/depl_nodes.xml]: In the Web UI: 'Username for Accessing NetApp' and 'Password for Accessing NetApp'

Comment (cjschroder) [inline xml/depl_nodes.xml]: Ok I'll look at them again. I took a quick peek to verify that they were multiple fields, and didn't pay much attention to the field labels.

**Comment by taroth21** [comment]
Thanks!

**Pull request merged**